src/alto-ui/Actions/__tests__/Actions.test.js

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import { mount } from 'enzyme';
+
+import Actions from '../Actions';
+import Dropdown from '../../Dropdown';
+import Group from '../../Group';
+
+jest.mock('../../Dropdown', () => {
+  return props => {
+    return (
+      <div className="Dropdown" onClick={() => props.onClick} onKeyDown={() => props.onClick} role="menu" tabIndex="0">
+        {props.items.map(action => {
+          return (
+            <button key={`key_${action.key}`} id={`key_${action.key}`} onClick={() => action.onClick || props.onClick()} />
```

Comment:
```suggestion
            <button key={`key_${action.key}`} id={`key_${action.key}`} onClick={action.onClick || props.onClick} />
```

Reply:
Refactored in "Refactor after code review" commit